blockchain/types/tx_internal_data.go

```diff
@@ -553,16 +553,35 @@ func toWordSize(size uint64) uint64 {
 }
 
 func IntrinsicGasPayload(gas uint64, data []byte, isContractCreation bool, rules params.Rules) (uint64, error) {
-	// Bump the required gas by the amount of transactional data
+	// // Bump the required gas by the amount of transactional data
```

Comment:
```suggestion
	// Bump the required gas by the amount of transactional data
```

Reply:
Uhm, I thought that I had added the comment requests. Can you let me know which statements were absent for comments in `IntrinsicGas()` header?

Reply:
I interpreted the meeting minutes like this:

```
func IntrinsicGas() {
	...
	if r.IsIstanbul {
		// tx types 1,2 only exist after istanbul; so they take this path.
		// tx types 8+ take this path as well.
		gasPayloadWithGas, err = IntrinsicGasPayload(gas, data, contractCreation, r)
	} else {
		// only for tx type 0 before istanbul.
		gasPayloadWithGas, err = IntrinsicGasPayloadLegacy(gas, data)
	}
}

// Klaytn-TxTypes since genesis, and EthTxTypes since istanbul use this.
func IntrinsicGasPayload() {...}

// Eth-TxTypes before istanbul use this. Before istanbul, there's only tx type 0.
func IntrinsicGasPayloadLegacy() {...}
```

Reply:
> Could you add the discussed comments? (for IntrinsicGas)

The above [comment](https://github.com/kaiachain/kaia/pull/107#discussion_r1798726357) is about this.


This [suggested change](https://github.com/kaiachain/kaia/pull/107#discussion_r1798699207) is an orthogonal issue. (`// //` should be `//`)

Reply:
> This https://github.com/kaiachain/kaia/pull/107#discussion_r1798699207 is an orthogonal issue. (// // should be //)

Yes I know. Just wanted to proceed conversation in a thread. Thank you for your kindness.


The comment below seems the opposite case. For now, I added it. Do you see this is a redundant one?
```
// tx types 1,2 only exist after istanbul; so they take this path.
		// tx types 8+ take this path as well.
```

Reply:
I'm ok without it